src/main/kotlin/ru/buhinder/alcopartyservice/util/Util.kt

```diff
@@ -0,0 +1,13 @@
+package ru.buhinder.alcopartyservice.util
+
+import org.springframework.stereotype.Component
+import ru.buhinder.alcopartyservice.dto.response.Pagination
+import kotlin.math.ceil
+
+@Component
+class Util {
+
+    fun createPagination(total: Long, page: Int, pageSize: Int): Pagination {
```

Comment:
This could be a reusable pagination service

fun createPagination(entities: Collection, page: Int, pageSize: Int): Mono<Pagination>{
**pagination calculations**
}